test/integ/axpy/src/main.cpp

```diff
@@ -97,7 +98,7 @@ struct VectorAddKernelTester
         using Val = float;
 
         // Create the kernel function object.
-        VectorAddKernel kernel;
+        AxpyKernel kernel;
 
         // Get the host device.
         auto devHost(alpaka::dev::cpu::getDev());
```

Comment:
I would suggest it, because when I was writing a alpaka code for the first time it was somehow confusing why the cpu device is treated in a special way. Same behavior vor all accelerators is less confusing I think. Furthermore, two methods for the same functionality raises the question which one to use. Is one better then the other ...